src/app/cli/src/app.rs

```diff
@@ -437,8 +437,11 @@ fn configure_logging(output_config: &OutputConfig, workspace_layout: &WorkspaceL
             "\n{}",
             console::style(
                 "Oh no, looks like kamu has crashed!\n\
+                Please follow the next install intructions https://docs.kamu.dev/cli/get-started/installation/. \n\
+                Also, please, run `kamu system diagnose` to check does your environment match the requirements. \n\
```

Comment:
Pointing to installation instructions after crash seems strange. I think only `diagnose` should do that if it detects some issues with the environment.

I suggest using this message:
```
Oh no, looks like kamu has crashed!
Please try running `kamu system diagnose` to verify your environment.
If the issue persists, help us by reporting this problem at https://github.com/kamu-data/kamu-cli/issues.
```

And also expanding @s373r's recent GH issue template to mention running `diagnose` command.